change/@microsoft-teams-js-24933635-de42-4b04-b53f-935c1799929a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "draft solution with optional apiVersion",
```

Comment:
This comment should be in the past tense and be written to communicate to external developers what has changed. Maybe something like "Added additional telemetry to `app`, `dialog`, <etc. etc.> capabilities"

Reply:
Updated. Please take another look.